app/assets/stylesheets/forever_style_guide/modules/_nav-account_dropdown.scss

```diff
@@ -104,6 +104,12 @@ $padding-small-horizontal: 3px;
   text-align: left;
 }
 
+.user_info-logged_in-account_dropdown-storage-name {
+  display: inline-block;
+  max-width: 200px; // Req for Safari
```

Comment:
prefer variable here

Reply:
I replaced with variable but prefer to keep comment so it is clear it is required for Safari, I feel like if testing in Chrome someone could think of this is not necessary without getting into cross-browser testing and I don't think adding the context of it being used for Safari in the variable name is appropriate either